public/components/datasources/components/manage/accelerations/acceleration_details_flyout.tsx

```diff
@@ -18,22 +18,19 @@ import {
 import React, { useEffect, useState } from 'react';
 import { AccelerationDetailsTab } from './flyout_modules/acceleration_details_tab';
 import { AccelerationSchemaTab } from './flyout_modules/accelerations_schema_tab';
-import { AccelerationSqlTab } from './flyout_modules/acceleration_sql_tab';
-import {
-  getRefreshButtonIcon,
-  onRefreshButtonClick,
-  onDiscoverButtonClick,
-  onDeleteButtonClick,
-} from './utils/acceleration_utils';
+import { onDiscoverIconClick, AccelerationActionType } from './utils/acceleration_utils';
 import { coreRefs } from '../../../../../framework/core_refs';
 import { OpenSearchDashboardsResponse } from '../../../../../../../../src/core/server/http/router';
 import { CachedAcceleration } from '../../../../../../common/types/data_connections';
+import { useAccelerationOperation } from './acceleration_operation';
+import { AccelerationActionOverlay } from './acceleration_action_overlay';
 
 export interface AccelerationDetailsFlyoutProps {
   index: string;
```

Comment:
Should we remove the `index` prop as discussed earlier? We get this already from `acceleration` prop

Reply:
No. I found out that this "index" is actually the displayed name we processed. For example if it is a skiiping index, it doesn't have the "indexName". So this `index` is actually referencing to the index we are showing in the rows and detail flyout. We cannot remove this.

Reply:
I think we need to have that as a function call inside the flyout too. Otherwise, we cannot expect workbench to reimplement the logic.

Reply:
Transferring the discussion with @dai-chen here: lets keep the skipping index name just as `skipping_index` and we add the full path of this skipping index in the pop up toast and title of overlay: ${datasource}.${database}.${table}.